### DISCUSSION TITLE
Question about Axway Supported Release

### DISCUSSION THREAD

**Discussion by lyr3051** [Axway-API-Management-Plus/apigateway-openlogging-elk#213] [Q&A]
Hi

Could you help me make sure what I understand below is correct or not?

1.Axway Supported release will release a new version quarterly just like APIM release?

2.As I know, if I create an issue in Github, you will fix the bug by releasing a new community release. 
But how will the bug be fixed if I create a normal Axway support cases?
Would it have a patch to fix the bug or release a new Axway Support release just like community release?

3.The latest Axway Supported release is 4.2.0 now , so it is the only Axway Supported release could be supported, the previous 4.0.2 and 4.0.1 one are no longer supported?

Thanks

**Comment by cwiechmann** [comment]
> 1.Axway Supported release will release a new version quarterly just like APIM release?

With every APIM-Release the latest available version of the ELK-Solution is tested and if it is a community version, it becomes the next official Axway release.

> 2.As I know, if I create an issue in Github, you will fix the bug by releasing a new community release.
But how will the bug be fixed if I create a normal Axway support cases?
Would it have a patch to fix the bug or release a new Axway Support release just like community release?

The difference is the Axway support policy with guaranteed reaction times, etc. 
Bugs will be fixed in the community version and as backports in the only available official release, which will be released as a patch version.

> 3.The latest Axway Supported release is 4.2.0 now , so it is the only Axway Supported release could be supported, the previous 4.0.2 and 4.0.1 one are no longer supported?

That’s correct. As the update is super simple, customers should upgrade to either the latest supported or the community version. Only these both versions are supported at a time.

Hope this helps,
Chris

**Reply by lyr3051** [reply]
Hi @cwiechmann 

> The difference is the Axway support policy with guaranteed reaction times, etc.Bugs will be fixed in the community version and as backports in the only available official release, which will be released as a patch version.

I still not very clear about it. 

Could you give an example if I find a bug in Axway Support release?

**Reply by lyr3051** [reply]
If I find a bug in Axway Support release 4.2.0, you will fix this bug in community 4.2.0 version, do I have to reload the Axway Support release 4.2.0 to get the patch version? or there will be a new version called patch 4.2.0 version I should download it?

**Reply by cwiechmann** [reply]
If the bug you found is critical, it will be fixed in the supported major version (4.2.0) by creating a new minor version (e.g. 4.2.1).

But keep in mind, only as long as there is no newer officially supported version. Once there is a newer supported version you have to upgrade to this version to get a fix.
